src/platform/site-wide/user-nav/components/SearchHelpSignIn.jsx

```diff
@@ -74,7 +74,7 @@ class SearchHelpSignIn extends Component {
         />
         <a
           className="vads-u-color--white vads-u-text-decoration--none vads-u-padding-top--0p5 vads-u-padding-x--1 vads-u-font-weight--bold"
-          href="/contact-us/"
+          href="https://www.va.go/contact-us/"
           onClick={() => recordEvent({ event: 'nav-jumplink-click' })}
```

Comment:
nit - this doesn't seem like the right GA event?

Reply:
No clue, not the issue I'm working on atm lol

Reply:
CC @jonwehausen is this the correct analytic event (`nav-jumplink-click`) for the `Contact Us` button in the top nav?